src/platforms/platform.dom.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
+var elementResizeDetectorMaker = require('element-resize-detector');
 
 // Chart.Platform implementation for targeting a web browser
 module.exports = function(Chart) {
 	var helpers = Chart.helpers;
+	var erd = null;
```

Comment:
[nit] I'd prefer a more descriptive name here since it will get minified away. Perhaps `elementResizeDetector`